ui/packages/upload/src/utils.ts

```diff
@@ -2,27 +2,19 @@ import type { FileData } from "./types";
 
 export function normalise_file(
```

Comment:
Cleaned up this file to avoid duplicate code using typscript overloading: https://www.tutorialsteacher.com/typescript/function-overloading

Reply:
I'm against function overloading and don't want us to introduce it here. This duplication is entirely unnecessary though and something that I missed in another PR (due to the file size issues). Will suggest an alternative.

Reply:
Feel free to push directly to the branch @pngwn